.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-    - image: circleci/golang:1.12
+    - image: circleci/golang:1.13.7
```

Comment:
Is this the right version?

Reply:
It's the same version as in Vault: https://github.com/hashicorp/vault/blob/master/Makefile#L22 I needed to upgrade to at least 1.13 because I'm using the `%w` formatter in `fmt.Errorf` which was introduced in 1.13.